packages/d2-app/src/commands/scripts/support/semantic-release-update-deps.js

```diff
@@ -0,0 +1,93 @@
+const fs = require('fs')
+const path = require('path')
+const SemanticReleaseError = require('@semantic-release/error')
+const AggregateError = require('aggregate-error')
+const normalizeAndValidatePackages = require('./normalizeAndValidatePackages')
+
+const verifyConditions = (config = {}, context) => {
+    const { silent, packages } = config
+    const { logger } = context
+    if (!packages || !packages.length || packages.lenght < 2) {
```

Comment:
`packages.lenght` should be `packages.length`

Reply:
Oof that's embarrassing.  Typescript would have caught this, but I'll settle for eagle-eyed @varl for now.